gondorapi/serializers/patient_data.py

```diff
@@ -58,4 +58,33 @@ def to_representation(self, instance):
             rep["patientSystolic"] = rep.pop("patient_systolic")
             rep["patientDiastolic"] = rep.pop("patient_diastolic")
             rep["patientWeightKg"] = rep.pop("patient_weight_kg")
-            return rep
\ No newline at end of file
+            return rep
+        
+    
+    class PatientDataCreateSerializer(serializers.ModelSerializer):
+        class Meta:
+            model = PatientData
+            fields = ["patient", "appointment" ,"created_by", "patient_systolic", "patient_diastolic", "patient_weight_kg", "clinician_notes", "is_notes_shared"]
```

Comment:
Do not pass in created_by, have this be based on the user submitting the request via passed in context